vnext/Folly/TEMP_UntilFollyUpdate/ConstexprMath.h

```diff
@@ -0,0 +1,968 @@
+/*
```

Comment:
https://github.com/facebook/folly/pull/2086 should hopefully land before next weeks Folly release, and also let us remove the fork of `Hash.h`

Reply:
It is great to see changes to be done in Folly repo!
Just nitpicking on your folly PR: did you consider using "if constexpr" for signed/unsigned instead of cloning the `constexpr_ipow` template.

Reply:
Folly targets C++ 14 as a minimum I believe.